install/kubernetes/helm/istio/values-istio-one-namespace-trust-domain.yaml

```diff
@@ -0,0 +1,22 @@
+# This is used to generate istio.yaml used for deprecated CI/CD testing.
+global:
+  # controlPlaneMtls enabled. Will result in delays starting the pods while secrets are
+  # propagated, not recommended for tests.
+  controlPlaneSecurityEnabled: true
+
+  mtls:
+    # Default setting for service-to-service mtls. Can be set explicitly using
+    # destination rules or service annotations.
+    enabled: true
+
+  ## imagePullSecrets for all ServiceAccount. Must be set for any clustser configured with private docker registry.
+  # imagePullSecrets:
+  #   - name: "private-registry-key"
+
+  # Default is 10s second
+  refreshInterval: 1s
+
+  trustDomain: test.local
```

Comment:
Does this value need to be in global?  e.g. can it be pushed to the security subchart's values.yaml?